dev/tools/gen_defaults/lib/input_chip_template.dart

```diff
@@ -30,7 +30,20 @@ class _${blockName}DefaultsM3 extends ChipThemeData {
   late final TextTheme _textTheme = Theme.of(context).textTheme;
 
   @override
-  TextStyle? get labelStyle => ${textStyle("$tokenGroup.label-text")};
+  TextStyle? get labelStyle => ${textStyle("$tokenGroup.label-text")}?.copyWith(
```

Comment:
Please follow the same style I've suggested for `FilterChip` template for this class and run generate command to update widget classes